supporter-product-data/cloudformation/cfn.yaml

```diff
@@ -450,16 +450,25 @@ Resources:
       AlarmDescription: >
         The supporter-product-data step function failed, check for failed executions here:
         https://eu-west-1.console.aws.amazon.com/states/home?region=eu-west-1#/statemachines/view/arn:aws:states:eu-west-1:865473395570:stateMachine:supporter-product-data-PROD
-      MetricName: ExecutionsFailed
-      Namespace: AWS/States
-      Dimensions:
-        - Name: StateMachineArn
-          Value: !Ref SupporterProductData
+      Metrics:
+        - Id: errors
```

Comment:
```suggestion
        - Id: errorsOutsideZuoraSlowHours
```

Reply:
comment below looks good, I wasn't sure where the ID is shown?  Might not be worth updating but a suggestion anyway